runtimes/php-8.1/server.php

```diff
@@ -3,96 +3,191 @@
 
 const USER_CODE_PATH = '/usr/code-start';
 
-function join_paths() {
-    $paths = array();
+class Response {
```

Comment:
We should prefix the class names with `Runtime` for consistency across runtimes

Reply:
Updated.